Doc/library/os.rst

```diff
@@ -3780,6 +3780,174 @@ features:
    .. versionadded:: 3.10
 
 
+.. function:: timerfd_create(clockid, flags, /)
+
+   Create and return a timer file descriptor (*timerfd*).
+   The file descriptor supports :func:`read` with a buffer size of 8,
+   :func:`~select.select`, and :func:`~select.poll`.
+
+   .. seealso:: The :manpage:`timerfd_create(2)` man page.
```

Comment:
The positioning of the ``seealso`` directives is inconsistent -- some are after the intro para and some are at the end.

Reply:
@AA-Turner 
Moved.